archunit/src/test/java/com/tngtech/archunit/lang/ArchConditionTest.java

```diff
@@ -226,6 +234,41 @@ public void double_never_or() {
         assertThat(events.containViolation()).as("Events contain violation").isFalse();
     }
 
+    @Test
+    public void from_predicate() {
+        JavaClass object = new ClassFileImporter().importClass(Object.class);
+
+        assertThat(ArchCondition.from(alwaysFalse().as("some description")))
+                .hasDescription("some description")
+                .checking(object)
+                .containViolations(String.format("Class <%s> does not satisfy some description in (%s.java:0)", Object.class.getName(), Object.class.getSimpleName()));
+
+        assertThat(ArchCondition.from(alwaysTrue().as("some description")))
```

Comment:
Is the predicate's description relevant in this case?
```suggestion
        assertThat(ArchCondition.from(alwaysTrue()))
```